src/main/java/org/hl7/davinci/priorauth/Database.java

```diff
@@ -246,11 +311,18 @@ public boolean write(String resourceType, Map<String, Object> data) {
     boolean result = false;
     if (data != null) {
       try (Connection connection = getConnection()) {
-        String sql = "INSERT INTO " + resourceType + " (" + setColumns(data.keySet()) + ") VALUES "
-            + setValues(data.values()) + ";";
-        PreparedStatement stmt = connection.prepareStatement(sql);
+        String valueClause = "";
+        for (int i = 0; i < data.values().size() - 1; i++)
+          valueClause += "?,";
+        valueClause += "?";
+
+        String sql = "INSERT INTO " + resourceType + " (" + setColumns(data.keySet()) + ") VALUES (" + valueClause
+            + ");";
+        Collection<Map<String, Object>> maps = new HashSet<Map<String, Object>>();
+        maps.add(data);
+        PreparedStatement stmt = generateStatement(sql, maps, connection);
         result = stmt.execute();
-        logger.info(sql);
+        // logger.info(stmt.toString()); // FOR DEBUG
```

Comment:
Maybe change to logger.debug() ?
We should add a method to easily change the logging level to debug from default info with a configuration or environment variable.

Reply:
I searched around but I couldn't get this to work easily so I added to PRIORAUTH-16 (which deals with sending the log to a file). For the time being I will leave it as info so if anything goes wrong at the Connectathon we will have all the information available